### ISSUE TITLE
[Feature request][1.16.1] Include FOV effects and Distortion effects in standardsettings

### ISSUE THREAD

**Issue by whelk0** [KingContaria/StandardSettings#35]
Would it be possible to add FOV effects and Distortion effects from the accessibility menu in standardsettings ? Currently they are reset whenever the minecraft instance is relaunched

<img width="1255" height="914" alt="Image" src="https://github.com/user-attachments/assets/87e12bee-8149-4a9d-bf2d-f2b0bf9fd104" />

**Comment by davidenwang** [comment]
Hmm any reason why this is not planned? Would also be interested in this feature

**Comment by maskersss** [comment]
> they are reset whenever the minecraft instance is relaunched

separate issue, you should be changing them in extraoptions settings and not accessibility settings 

i don't see why youd need to have them set to something every time when a new world is created

**Comment by davidenwang** [comment]
> > they are reset whenever the minecraft instance is relaunched
> 
> separate issue, you should be changing them in extraoptions settings and not accessibility settings
> 
> i don't see why youd need to have them set to something every time when a new world is created

Ahhh i didn't realize these settings were backported from later versions of the games. I was just interested in them because I prefer playing on less FOV effects so I end up changing the setting every time I launch the game. That works for now though
